api/events/events.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package events;
+
+option go_package = "github.com/talos-systems/siderolink/api/events";
+
+import "google/protobuf/any.proto";
+
+message EventRequest {
+  google.protobuf.Any data = 1;
+  string id = 2;
+}
+
+message EventResponse {}
+
+service EventSinkService {
+  rpc Publish(stream EventRequest) returns(EventResponse);
```

Comment:
my concern with streaming implementation here is that we would have no way to know that some events were delivered (from the Talos side). we just send the events and _hope_ that they got delivered

if we do a single-shot (unary) call, we have a response back immediately, and we know we should retry or we know that all events got pushed

Reply:
yeah... makes sense. I've updated